### ISSUE TITLE
How do I receive data from server?

### ISSUE THREAD

**Issue by mron** [segabor/OSCCore#24]
The SuperColliderClient example shows
`// get and print out frequency number from SuperCollider
let getFrqMessage = OSCMessage(address: "/s_get", args: [synthID, "freq"])
try channel.writeAndFlush(getFrqMessage, target: remoteAddr)`

I don't see how to get and process the response. How is it done?

**Comment by segabor** [comment]
The mentioned example consumes packets received from server in `debugOSCPacket()` function. Technically this block
```swift
  let bootstrap = DatagramBootstrap(group: group)
      .channelOption(ChannelOptions.socket(SocketOptionLevel(SOL_SOCKET), SO_REUSEADDR), value: 1)
      .channelInitializer { channel in
          channel.pipeline.addHandlers([OSCPacketReader(), OSCDebugHandler()])
  }
```
sets up the pipeline where the last chain `OSCDebugHandler` calls `debugOSCPacket` with the unwrapped packet.

**Comment by mron** [comment]
Thanks for replying! What port is it listening on?

**Comment by segabor** [comment]
According to below network configuration
```swift
let channel = try bootstrap.bind(host: "127.0.0.1", port: 57150).wait()
let remoteAddr = try SocketAddress(ipAddress: "127.0.0.1", port: 57110)
```
the client is expecting responses on port **57150**.

**Comment by mron** [comment]
Thanks,
That's what I expect. I'm not seeing any incoming data at the client. I'll keep looking into this.

**Comment by mron** [comment]
I got it to work. I had to change:
`(cxt: ` to `(context: `

**Comment by segabor** [comment]
Awesome!

**Comment by segabor** [comment]
Thanks for discovering the issue! I fixed it (see #25)
